format/Schema.fbs

```diff
@@ -178,6 +178,9 @@ table FixedSizeBinary {
 table Bool {
 }
 
+table RunEndEncoded {
```

Comment:
Can you add a short comment above to briefly explain this?

Reply:
Added